content/docs/data_pipeline/introduction/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+weight: 1
+title: "Introduction"
+---
+
+# Introduction
+
+**This is work in progress.**
+
+The FAIR Data Pipeline (FDP) is intended to enable tracking of provenance of data used in epidemiological modelling. File APIs written in C++, Java, Julia, Python and R can be called by model software for data insgestion. These interact with a local relational database storing metadata and the local filesystem, and are configured by a `yml` file associated with the model run. Local files and metadata are synchronised with a remote registry via a command line tool (`fdp`).
+
+The key benefits of using the FAIR Data Pipeline are:
+
+- Provenance of model outputs is tracked to inputs
+- Multiple language support
+- Designed to run on a broad range of platforms (including HPC)
+- Access to shared central data for covid modelling
+
+## Running Models
+
+To use the FAIR Data Pipeline with a piece of modelling software, you must add a language specific API as a dependency and interact with data registered in the pipeline via the methods it presents. Each model run must be configured using a [`config.yml`](../interface/_index.md) file which specifies inputs and outputs by metadata.
```

Comment:
To be explicit and match the diagram below, do you want to say "you must add a language specific **File** API"? or perhaps "Local API"? (do you need both?) Is this the terminology we're using, not "Modelling API" or something else?

Reply:
We originally used Standard API, and contrasted it with a File API as the low-level interface. I'd prefer to drop both now as too vague... I have been calling it the Modelling API, but that's wrong too (not always modelling). Maybe we call it the Pipeline API now? Then there is a separate Registry API, which is what the Pipeline API uses to interact with the registry.

Reply:
Thanks.